### ISSUE TITLE
Loss display during epoch

### ISSUE THREAD

**Issue by akum04** [PyTorchLightning/pytorch-lightning#3410]
###  During the training, I'm using the custom loss function to train my model. However the loss are displayed as 0.000, but when I display the same value to display as different variable it gives 4.73e-5 (some value in exponential format). 

Epoch 80:  10%|██▌                       | 100/1013 [01:33<14:11,  1.07it/s, loss=0.000, v_num=None, train_loss=4.73e-5]

both loss and train_loss display the same value. why one displays in exponential format and other doesn't.

### Is it  possible does it prohibits the model from converging. because when i use the same parameters to train the model in normal way it converges, however with the pytorch lightning the model doesn't converge beyond certain limit.
